js/notify/electron-notify.js

```diff
@@ -208,12 +227,32 @@ function calcDimensions() {
 function setupConfig() {
     closeAll();
 
-    // Use primary display only
-    let display = electron.screen.getPrimaryDisplay();
+    // This feature only applies to windows
+    if (!isMac) {
+        let screens = electron.screen.getAllDisplays();
+        if (screens && screens.length >= 0) {
+            externalDisplay = screens.find((screen) => {
+                let screenId = screen.id.toString();
+                return ((screen.bounds.x !== 0 || screen.bounds.y !== 0) && screenId === notfDisplay);
+            });
+        }
+    }
 
-    config.corner = {};
-    config.corner.x = display.bounds.x + display.workArea.x;
-    config.corner.y = display.bounds.y + display.workArea.y;
+    let display;
+    // Update the notification to display on the user selected screen
+    if (externalDisplay) {
+        // use user selected display
+        display = externalDisplay;
+        config.corner = {};
+        config.corner.x = display.bounds.x;
+        config.corner.y = display.bounds.y;
+    } else {
+        // Use primary display
+        display = electron.screen.getPrimaryDisplay();
+        config.corner = {};
+        config.corner.x = display.bounds.x + display.workArea.x;
```

Comment:
don't we also want the same value here for external display?
that is code could be simplified:
```
config.corner = {}
config.corner.x = display.bounds.x + display.workArea.x;
config.corner.y = display.bounds.y + display.workArea.y;
let display = externalDisplay ? externalDisplay : electron.screen.getPrimaryDisplay();
```